code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -68,7 +67,7 @@
 	origin_tech = list(TECH_BLUESPACE = 4)
 	icon_state = "holdingpack"
 	max_w_class = ITEM_SIZE_LARGE
-	max_storage_space = 56
```

Comment:
this probably doesnt make a difference since bags of holding have that override, but better safe than sorry